grails-app/services/org/pih/warehouse/core/DashboardService.groovy

```diff
@@ -175,7 +183,7 @@ class DashboardService {
      * @param threshold the threshold filter
      * @return a list of inventory items
      */
-    List getExpiringStock(Category category, Location location, String expirationStatus) {
+    List getExpiringStock(Category category, Location location, String expirationStatus, Date startDate, Date endDate) {
```

Comment:
just a comment - 4 arguments feels a lot to me, I think in the near future, we should rely on command classes more, since they give us more flexibility and better dev experience in my opinion.